modules/server_new/src/test/scala/observe/server/ObserveEngineSuite.scala

```diff
@@ -362,7 +362,7 @@ class ObserveEngineSuite extends TestCommon {
     val resources: Set[Resource | Instrument] = Set(Instrument.GmosNorth, TCS)
 
     val obsTypes: NonEmptyList[(ObserveClass, StepConfig)] = NonEmptyList(
-      (ObserveClass.ProgramCal, StepConfig.Dark),
+      (ObserveClass.NightCal, StepConfig.Dark),
```

Comment:
Are darks taken at night?  If it was `ProgramCal` before it should be `NightCal` now so :+1:

Reply:
I think darks can be taken at any time

Reply:
The test in particular doesn't care too much about the class, it doesn't check the content of keywords